src/main/resources/com/google/api/codegen/go/main.snip

```diff
@@ -227,7 +227,7 @@
                      tokenZeroValue = context.zeroValue(pageStreaming.getRequestTokenField().getType)
 
                     {@methodComment(method, methodName)}
-                    func (client *Client) {@methodName}(ctx context.Context, req {@inTypeName}, opts ...gax.CallOption) (*{@iteratorTypeName}, error) {
+                    func (client *Client) {@methodName}(ctx context.Context, req {@inTypeName}, opts ...gax.CallOption) *{@iteratorTypeName} {
```

Comment:
Not for this PR, perhaps, but I would like to see shorter, more idiomatic variable names:

s/client/c/
s/iterator/it/ (or iter)